examples/AspNetCore/OData/SomeODataOpenApiExample/Program.cs

```diff
@@ -64,7 +64,9 @@
 // Configure the HTTP request pipeline.
 
 app.UseSwagger();
-app.UseSwaggerUI(
+if ( app.Environment.IsDevelopment() )
+{
+    app.UseSwaggerUI(
     options =>
```

Comment:
nit: the lambda didn't indent to retain the same level of indentation (as you did in other places 😉 )